src/pages/cli/graphql/override.mdx

```diff
@@ -128,3 +128,49 @@ You can override the following @predictions directive resources that Amplify gen
 |[identifyTextAccess](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html)|IAM policy to enable Identify Text|
 |[identifyLabelsAccess](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-policy.html)|IAM policy to enable Identify Text|
 
+## Place AppSync Resolvers in Custom-named Stacks
+
+If you have a particularly large GraphQL schema, you may run into issues with too many resources defined in a stack. The most common case where this happens is in the ConnectionStack which contains the resolvers for all of the relational directives in the schema.
```

Comment:
Could you please add a line explaining that all the child stacks would still be part of the same root stack, so that customers are aware that this may not help any root-stack level limits